addons/test_mail/tests/test_message_track.py

```diff
@@ -42,6 +42,7 @@ def test_message_track_no_subtype(self):
         # one new message containing tracking; without subtype linked to tracking, a note is generated
         self.assertEqual(len(self.record.message_ids), 1)
         self.assertEqual(self.record.message_ids.subtype_id, self.env.ref('mail.mt_note'))
+        self.assertEqual(self.record.message_ids.author_id, self.partner_employee)
```

Comment:
Just for  :socks: would test author_id before subtype_id (to avoid random field check, we choose to do them alphabetically). Same for other asserts below :)